src/repositories/video/video.repository.ts

```diff
@@ -120,17 +96,13 @@ export class VideoRepository {
     tags: string[];
     community: string;
     language: string;
-    videoUploadLink: string;
     beneficiaries: string;
-    permlink: string;
   }): Promise<Video> {
     return await this.videoModel.create({
-      video_id,
```

Comment:
I assume the video_id and permlink generation is moved into `videoModel.create()`

Reply:
Yep, setting them to `default: () = > uuid()` this process becomes automated.

Reply:
Can you point out the line number? I did not see that in this PR

Reply:
https://github.com/spknetwork/acela-core/pull/21/files#diff-209cc68f61fdd50bb2bcd2e7d03e86c0c84ba8c440488bdb44f9e7d4cf20e601R10

Reply:
Cool thx